Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rust:1.44-slim-stretch
+FROM rust:1.45-slim-stretch
```

Comment:
Bumped to 1.45 because `strip_prefix` and `strip_suffic` on String's were not stabilized in 1.44